episode-10-switching-aws-to-roar/app/representers/product_representer.rb

```diff
@@ -0,0 +1,30 @@
+require 'roar/decorator'
+# require 'roar/json/json_api'
```

Comment:
So far JSON-API is not output, just standard JSON. I tried to require this file, but it complains:

```
Failures:

  1) ProductsController GET /products/{id} when product exists will retrieve record in valid JSON-API format
     Failure/Error: require 'roar/json/json_api'

     NameError:
       uninitialized constant Roar::JSON::JSONAPI
     # ./app/representers/product_representer.rb:2:in `<top (required)>'
     # ./app/controllers/products_controller.rb:17:in `show'
     # ./spec/support/default_format.rb:12:in `block in <module:RequestHelpersCustomized>'
     # ./spec/requests/products_controller_spec.rb:58:in `block (4 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # ActionView::MissingTemplate:
     #   Missing template products/show, application/show with {:locale=>[:en], :formats=>[:json], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby]}. Searched in:
     #     * "/Users/psturgeon/src/livecoding-apisyouwonthate/episode-10-switching-aws-to-roar/app/views"
     #   ./app/controllers/products_controller.rb:17:in `show'
```